### PR TITLE
Uses `python3 -m pip` instead of calling bare `pip`

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

File: variables.tf [hunk 1]
```diff
@@ -28,7 +28,7 @@ variable "tfi_assign_public_ip" {
 }
 
 variable "tfi_win_instance_type" {
-  default = "t2.medium"
+  default = "t2.large"
```
Comment (YakDriver) [inline variables.tf]: Since CodeBuild sets this via env, this doesn't have much effect.

**Pull request merged**